manim/mobject/types/vectorized_mobject.py

```diff
@@ -2022,6 +2022,43 @@ def __init__(self, vmobject, **kwargs):
 
 
 class DashedVMobject(VMobject, metaclass=ConvertToOpenGL):
+    """
```

Comment:
We should have a one sentence description of the class here.

Reply:
How about this for a description?
```suggestion
    """A :class:`VMobject` composed of dashes instead of lines.
```